meta-zarhus-bsp/meta-zarhus-bsp-x86_64/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -0,0 +1,6 @@
+# Ensure the ESP is always mounted by label, not by device path
+do_install:append() {
+    # Add our deterministic entry
+    echo 'LABEL=zarhus-boot  /boot  vfat  defaults,noatime  0  0' \
```

Comment:
What's the advantage of this rather than providing the whole static fstab file for our distro?

I provided you example from meta-otab in our chat.